docs/security/trust/content_trust.md

```diff
@@ -0,0 +1,262 @@
+creators want to iterate over a image before signing it's tag officially.
+
+Image consumers can enable trust to ensure that images they use were signed. If
+a consumer enables trust, they can only pull or run trusted images. Enabling
+trust is like wearing a pair of rose-colored glasses. Consumers "see" only
+signed images tags and the less desirable, unsigned image tags are "invisible" to them.
+
+![Trust view](../images/trust_view.png)
+
+To the consumer who is not using trust, every image is visible and can be consumed.
+
+
+### Trust operations and keys
+
+When trust is enabled, `docker` CLI commands that operate on tagged images must
+either have content signatures or explicit content hashes. The commands that
+operate with trust are:
+
+* `push`
+* `build`
+* `create` 
+* `pull`
+* `run` 
+
+Trust for an image tag is managed through the use of keys. Docker's content
+trust makes use four different keys: 
+
+| Key                 | Description                                                                                                                                                                                                                                                                                                                                                                         |
+|---------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| offline key            | Root of trust for a image tag. When trust is enabled, you create this key once. Then, you can use this key to create tagging keys. |
+| target and snapshot | These two keys are known together as the "repository" key. When trust is enabled, you create this key when you add a new image repository. If you have the offline key, you can export the tagging key and allow other images creator that sign image tags.    |
+| timestamps          | This key applies to a repository. It allows Docker repositories to have freshness security guarantees without requiring periodic content refreshes on the client's side. Each tag is signed with a tagging key and a timestamp key.                                                                                                             |
+
+With the exception of timestamps, all the keys are generated and stored locally
+client-side. The timestamp is generated and stored in a Docker registry.
+
+The following image depicts the various keys and their relationships:
+
+![Trust components](../images/trust_components.png)
+
+>**WARNING**: Loss of the offline key is **very difficult** to recover from.
+>Correcting this loss requires intervention from [Docker
+>Support](https://support.docker.com) to reset the image tag state. This loss
+>also requires **manual intervention** from every consumer that used the tagged
+>image prior to the loss. You should backup this key somewhere safe. 
```

Comment:
`You should backup this key somewhere safe, and given that it will only be required to create new repositories, we strongly recommend that you store it offline`